### PR TITLE
Compile.html.md update

### PR DESCRIPTION
**Pull request by SLASHLogin**
Changed files directories. Added link for openSSL precompiled version. Added spaces in lines 49 44 jsut like you have done. Added section, and few more things. Sorry I'm to lazy to list them, anyway you should check code for any mistakes made by me

### CODE REVIEW

**Comment by SLASHLogin** [comment]
Don't merge atm

**Comment by SLASHLogin** [comment]
Now its good, idk what was wrong but it works

File: source/compile.html.md [hunk 1]
```diff
@@ -17,14 +17,18 @@ search: true
 The first step to using Sleepy Discord
 </aside>
 
+##Requirements
+
+- Visual Studio
```
Comment (yourWaifu) [inline source/compile.html.md]: Visual studio isn't required

File: source/compile.html.md [hunk 2]
```diff
@@ -56,34 +67,40 @@ And in the cpr folder, open the opt folder. You should see an empty folder named
...
 <aside class="notice">
 Sleepy Discord uses version 1.10.8, and does not work with 1.10.6. [The latest version can be downloaded here](https://sourceforge.net/projects/asio/files/latest/download)
 </aside>
 
-Last, you are also going to need openSSL. That can be found here [https://www.openssl.org/](https://www.openssl.org/). Extract the folder inside to somewhere to work on it. To install look at the instructions in INSTALL because this has its own set of instructions including downloading and installing some other dependencies. However, you can find openSSL precompiled on the internet. Once that's done, merge (or move) the include folder with the include folder in deps. Also (I forgot to do this for the pictures) you'll need to place the lib folder into deps, you'll need this to link Sleepy Discord. 
+Last, you are also going to need openSSL. There's two options:
+1. Download precompiled openSSL [https;//github.com/David-Reguera-Garcia-Dreg/Precompiled-OpenSSL-Windows/archive/master.zip](https;//github.com/David-Reguera-Garcia-Dreg/Precompiled-OpenSSL-Windows/archive/master.zip).
```
Comment (yourWaifu) [inline source/compile.html.md]: I think those links are better. https://wiki.openssl.org/index.php/Binaries https://www.npcglib.org/~stathis/blog/precompiled-openssl/. However, you'll need 7-Zip to unzip them

**Comment by yourWaifu** [comment]
Looks pretty good, however, I would like some small changes to be made.

**Comment by SLASHLogin** [comment]
Yeah no problem I will change that and then send next commit.

File: source/compile.html.md (hunk 2 above)
Comment (SLASHLogin) [reply source/compile.html.md]: Or winrar or other program like that, yeah?

File: source/compile.html.md (hunk 1 above)
Comment (SLASHLogin) [reply source/compile.html.md]: at least I can add (reccomended)

**Comment by SLASHLogin** [comment]
I were also thinking about just doing link to gdrive to download compiled version to skip using .py and doing everything manually.

**Comment by yourWaifu** [comment]
Thank you!

**Comment by SLASHLogin** [comment]
No problem! I think that I should help you with code at least changing site. Anyway I will take look at other pull request about docs. And yeah just keep eye on site as I can.

**Comment by SLASHLogin** [comment]
Just to sum up just make even empty pull request with comment to me so I will take look at this what you want

**Pull request merged**